File: P006/main.java
import java.lang.*;

public class main {

    private static void findDifference(int n) {
        int result = 0;

        int squared = (int)(2*Math.pow(n, 3) + 3*Math.pow(n, 2) + n)/6;
        int addSquared = (int)Math.pow((n*(n+1))/2, 2);

        result = addSquared-squared;
        System.out.println(addSquared + " - " + squared + " = " + result);
    }

    public static void main(String[] args) {
        long startTime = System.currentTimeMillis();

        findDifference(100);

        long endTime = System.currentTimeMillis();
        long totalSpan = System.currentTimeMillis();
        System.out.println("Completed in " + totalSpan + " milliseconds.");
    }
}

File: P006/main.py
import time
import math

start = time.time()

def findDifference(n):
    result = 0

    squared = (2*pow(n, 3) + 3*pow(n, 2) + n)/6
    addSquared = pow((n*(n+1))/2, 2)

    result = addSquared-squared
    print str(addSquared) + " - " + str(squared) + " = " + str(result)

findDifference(100)

elapsed = time.time()

print("found in %s seconds") % (elapsed)
